src/Io/KeyboardInputHandler.cpp

```diff
@@ -317,6 +318,10 @@ void KeyboardInputHandler::GenerateGameplayActions() {
             engine->config->settings.AlwaysRun.Toggle();
             break;
 
+        case InputAction::Escape:
```

Comment:
Drop?

Reply:
Could drop, but I left it there as it should really be handled there in the end and not globally in GameWindowHandler::OnKey. To make that happen every close button should be hunted and it's InputAction::Invalid replaced with InputAction::Escape, also for some rare cases like credits menu there should be added invisible button with InputAction::Escape